script/DeployQwManager.s.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import {QWManager} from 'contracts/QWManager.sol';
+import {QWRegistry} from 'contracts/QWRegistry.sol';
+import {Script} from 'forge-std/Script.sol';
+
+contract DeployQwManager is Script {
+  QWManager public qwManager;
+  QWRegistry public qwRegistry;
+
+  function run() public {
+    vm.startBroadcast();
+
+    // Deploy QwManager
```

Comment:
We should add comment on top of this file and mention which all contracts are getting deployed here. (QWManager and QWRegistry)

Reply:
sure, will do